server/datastore/mysql/statistics.go

```diff
@@ -84,6 +84,12 @@ func (ds *Datastore) ShouldSendStatistics(ctx context.Context, frequency time.Du
 		stats.VulnDetectionEnabled = appConfig.VulnerabilitySettings.DatabasesPath != ""
 		stats.SystemUsersEnabled = appConfig.Features.EnableHostUsers
 		stats.HostsStatusWebHookEnabled = appConfig.WebhookSettings.HostStatusWebhook.Enable
+		stats.MDMEnabledAndConfigured = appConfig.MDM.EnabledAndConfigured
+		stats.HostExpiryEnabled = appConfig.HostExpirySettings.HostExpiryEnabled
+		stats.JiraIntegrationEnabled = len(appConfig.Integrations.Jira) > 0
+		stats.ZendeskIntegrationEnabled = len(appConfig.Integrations.Zendesk) > 0
```

Comment:
Same comment as Jira. I think we cut this one entirely for now and come back to this one.